### PR TITLE
Update Transfer Ownership instructions

### PR DESCRIPTION
**Pull request by vipulgupta2048**
With organizations released, instructions were needed to be updated in accordance with the terms used and new rules applied.

Reference: https://www.flowdock.com/app/rulemotion/s-sales/threads/jEaIKEFwC1JiIHC_4rQymv463CU
Change-type: patch
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

### CODE REVIEW

File: pages/learn/manage/actions.md [hunk 1]
```diff
@@ -122,14 +122,14 @@ You may also enable or disable public URLs for a subset of devices by selecting
 
 ### Transfer Application Ownership
 
-You can transfer application ownership between balenaCloud user accounts. In order to transfer application ownership, you will need to perform a series of steps using both the **source** account and **target** account.
+You can transfer application ownership between balenaCloud organizations. In order to transfer application ownership, you will need to perform a series of steps using both the **source** account's organization and **target** account's organization.
```
Comment (dfunckt) [inline pages/learn/manage/actions.md:125]:
```suggestion
Applications with all their associated devices, releases and members can be transferred to any other balenaCloud organization. Application transfers are between a **source** and a **target** organization.

Only organization administrators can initiate and complete application transfers. You must coordinate with one of the receiving organization's administrators to perform the following actions:
```

File: pages/learn/manage/actions.md [hunk 2]
```diff
@@ -122,14 +122,14 @@ You may also enable or disable public URLs for a subset of devices by selecting
...
 
-1. Take note of the Application name and balenaCloud username (*in the top-right drop-down*) in the **source** account
-2. In the **target** balenaCloud account, create a new balenaCloud Application using the same Application name
-3. In the **Members** of the Application you just created, add the **source** balenaCloud username as a **Developer**
+1. Take note of the Application name and balenaCloud username (*in the top-right drop-down*) in the **source** account.
```
Comment (dfunckt) [inline pages/learn/manage/actions.md:127]:
```suggestion
1. Take note of the application name in the **source** account.
```

File: pages/learn/manage/actions.md [hunk 3]
```diff
@@ -122,14 +122,14 @@ You may also enable or disable public URLs for a subset of devices by selecting
...
+2. In the **target** balenaCloud organization, create a new balenaCloud Application using the same Application name.
+3. In the **Members** of the Application you just created, add the **source** balenaCloud username as a **Developer** role. If you are an admin of the **target** organization, then no action is needed since you already have access to the new application.
 4. In the **source** balenaCloud account, select **<Application>** --> **Actions** --> **Transfer This Application**
```
Comment (dfunckt) [inline pages/learn/manage/actions.md:130]:
```suggestion
1. Take note of the application name in the **source** organization and your balenaCloud username (*in the top-right drop-down*).
2. Ask an administrator of the **target** balenaCloud organization to create a new application using the same application name.
3. Ask the administrator of the **target** balenaCloud organization to add you as a member of the newly created application with a Developer role, using your username. See [Adding members to an application](i-dont-know-the-url). If you are an administrator of the **target** organization, you already have access to the new application and you can skip this step.
4. In the **source** organization, select **<Application>** --> **Actions** --> **Transfer This Application** and pick the **target** organization from the list to complete the transfer.
```

File: pages/learn/manage/actions.md (hunk 2 above)
Comment (dfunckt) [reply pages/learn/manage/actions.md:127]: Ignore this comment, I couldn't get Github to delete it.

**Comment by vipulgupta2048** [comment]
@dfunckt there's this video in the docs for explaining transfer ownership, should we remove it as it would end up confusing people. The UI is fairly old. 

> You can watch [this video](https://www.youtube.com/watch?v=nkYve7PwqmE) to learn how to transfer application ownership.

File: pages/learn/manage/actions.md [hunk 4]
```diff
@@ -122,14 +122,16 @@ You may also enable or disable public URLs for a subset of devices by selecting
...
+1. Take note of the application name in the **source** organization and your balenaCloud username (*in the top-right drop-down*).
+2. Ask an administrator of the **target** balenaCloud organization to create a new application using the same application name.
+3. Ask the administrator of the **target** balenaCloud organization to [add you as a member][add-application-member] of the newly created application with a [`Developer`][developer] role, using your username. If you are an administrator of the **target** organization, you already have access to the new application & this step can be skipped.
+4. In the **source** organization, select **<Application>** --> **Actions** --> **Transfer This Application** and pick the **target** organization from the list to complete the transfer.
+
+__Note:__ If the **Transfer This Application** button is grayed out, ensure that you have created an empty application in the **target** organization with the same name as the source application, and that the **source** username has been added as a **Developer** to the **target** application.
```
Comment (thgreasi) [inline pages/learn/manage/actions.md:134]: > that the **source** username has been added as a **Developer** to the **target** application.

This sounds a bit confusing in the orgs era. It's more like:
> the user that is going to initiate the transfer from the **source** organization has been....

File: pages/learn/manage/actions.md [hunk 5]
```diff
@@ -122,14 +122,16 @@ You may also enable or disable public URLs for a subset of devices by selecting
...
-2. In the **target** balenaCloud account, create a new balenaCloud Application using the same Application name
-3. In the **Members** of the Application you just created, add the **source** balenaCloud username as a **Developer**
-4. In the **source** balenaCloud account, select **<Application>** --> **Actions** --> **Transfer This Application**
+Only organization [administrators][administrator] can initiate and complete application transfers. You must coordinate with one of the receiving organization's administrators to perform the following actions:
 
-__Note:__ If the **Transfer This Application** button is grayed out, ensure that you have created an empty Application in the **target** balenaCloud account with the same name as the source Application, and that the **source** username has been added as a **Developer** to the **target** Application.
+1. Take note of the application name in the **source** organization and your balenaCloud username (*in the top-right drop-down*).
+2. Ask an administrator of the **target** balenaCloud organization to create a new application using the same application name.
```
Comment (thgreasi) [inline pages/learn/manage/actions.md:130]: NIT: Even though "empty" is also mentioned in the notes, imo it's just a single word and I see no reason to not add it in here as well. Not sure whether this was discussed before though.
```suggestion
2. Ask an administrator of the **target** balenaCloud organization to create a new empty application using the same application name.
```

**Review by thgreasi — APPROVED** [review]
:+1:

**Pull request merged**